languages/altapay-pl_PL.po

```diff
@@ -0,0 +1,152 @@
+"Report-Msgid-Bugs-To: https://wordpress.org/support/plugin/altapay-for-woocommerce\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: pl_PL\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"POT-Creation-Date: 2024-07-18T10:05:00+00:00\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"X-Generator: WP-CLI 2.10.0\n"
+"X-Domain: altapay\n"
+
+
+#: altapay.php:254
+msgid "AltaPay Payment Actions"
+msgstr "Działania płatnicze AltaPay"
+
+#: altapay.php:297
+#: classes/core/AltapayOrderStatus.php:52
+#: classes/core/AltapaySettings.php:63
+#: classes/core/AltapaySettings.php:437
+#: helpers/traits/AltapayMaster.php:69
+#: terminals/Altapay_Test_Terminal.class.php:418
+msgid "Could not connect to AltaPay!"
+msgstr "Nie można połączyć się z AltaPay!"
+
+#: altapay.php:311
+#: classes/core/AltapayOrderStatus.php:62
+msgid "Could not fetch Payments from AltaPay!"
+msgstr "Nie można pobrać płatności z AltaPay!"
+
+#: altapay.php:360
+#: altapay.php:971
+msgid "Payment Released."
+msgstr "Płatność zwolniona."
+
+#: altapay.php:385
+msgid "Order got no transaction"
+msgstr "Zamówienie nie ma transakcji"
+
+#: altapay.php:708
+msgid "Payment Captured."
+msgstr "Płatność zrealizowana."
+
+#: altapay.php:832
+msgid "Payment refund is in progress."
+msgstr "Zwrot płatności jest w toku."
+
+#: altapay.php:915
+msgid "Payment Refunded."
+msgstr "Płatność zwrócona."
+
+#: views/tables/capture.blade.php:16
+#: views/tables/refund.blade.php:16
+msgid "Product name"
+msgstr "Nazwa produktu"
+
+#: views/tables/capture.blade.php:17
+#: views/tables/refund.blade.php:17
+msgid "Price with tax"
+msgstr "Cena z podatkiem"
+
+#: views/tables/capture.blade.php:18
+#: views/tables/refund.blade.php:18
+msgid "Price without tax"
+msgstr "Cena bez podatku"
+
+#: views/tables/capture.blade.php:19
+#: views/tables/refund.blade.php:19
+msgid "Ordered"
+msgstr "Zamówione"
+
+#: views/tables/capture.blade.php:20
+#: views/tables/refund.blade.php:20
+msgid "Discount Percent"
+msgstr "Procent rabatu"
+
+#: views/tables/capture.blade.php:21
+#: views/tables/refund.blade.php:21
+msgid "Quantity"
+msgstr "Ilość"
+
+#: views/tables/capture.blade.php:22
+#: views/tables/refund.blade.php:22
+msgid "Total amount"
+msgstr "Łączna kwota"
+
+#: views/tables/index.blade.php:59
+#: views/tables/index.blade.php:95
+msgid "Send order lines"
+msgstr "Wyślij linie zamówienia"
+
+#: views/tables/index.blade.php:66
+#: views/tables/index.blade.php:101
+msgid "Amount"
+msgstr "Kwota"
+
+#: views/tables/index.blade.php:68
+#: views/tables/index.blade.php:139
+#: views/tables/index.blade.php:143
+msgid "Capture"
+msgstr "Zrealizuj"
+
+#: views/tables/index.blade.php:103
+#: views/tables/index.blade.php:144
+#: views/tables/index.blade.php:148
+msgid "Refund"
+msgstr "Zwrot"
+
+#: views/tables/index.blade.php:113
+msgid "Transaction ID"
+msgstr "ID transakcji"
```

Comment:
Possibly to use full polish translation (`ID` abbreviation is not commonly used in Polish apart from people knowing English):

```
Identyfikator transakcji
```